src/bin/whirlwind.js

```diff
@@ -3,5 +3,6 @@
 const fs = require('fs');
 const nodeController = require('./../lib/node-controller');
 
-const config = JSON.parse(fs.readFileSync('whirlwind.json', 'utf8'));
+const filename = process.argv.length > 2 ? process.argv[process.argv.length - 1] : 'whirlwind.json';
```

Comment:
Add support for a first argument with the input json file location, if its not used, use the default `whirlwind.json`.